src/resolvers/Chat/messages.ts

```
@@ -1,5 +1,6 @@
-import { ChatMessage } from "../../models";
-import type { ChatResolvers } from "../../types/generatedGraphQLTypes";
+import { ChatMessage, InterfaceChatMessage } from "../../models";
```

ESLint

[error] Imports "InterfaceChatMessage" are only used as type.

(@typescript-eslint/consistent-type-imports)